pkg/ndctl/region.go

```diff
@@ -199,8 +199,11 @@ func (r *region) CreateNamespace(ctx gocontext.Context, opts CreateNamespaceOpts
 	align := mib2
 	namespacealign := align * r.InterleaveWays()
 	regionalign := r.GetAlign()
-	if regionalign == 0 {
-		regionalign = 1
+	if regionalign <= 1 {
+		// This fallback turned out to be necessary when emulating PMEM in
+		// libvirt (OpenShift 4.8 beta): both PMEM-CSI and ndctl failed
+		// to create a namespace of size 100MiB, whereas 96MiB worked.
+		regionalign = 16 * 1024 * 1024
```

Comment:
16MiB.

Reply:
I'm wondering whether we should be even more conservative and fall back to 96MiB. That's what was needed for https://github.com/intel/pmem-csi/issues/964

Reply:
I've switched to 96MiB and also refactored the code so that both namespace creation and capacity retrieval use the same helper function. We had similar (but not quite the same!) code in both.

Reply:
After going through issue #964, I feel like using 'no. of interleaves * 16MiB' as alignment is a good solution than fixing the alignment to 96MiB.

Reply:
But using that as alignment was exactly the problem in #964 - we did that and ignored the region alignment, which then didn't work.

The lesson from #964 is that we have to consider *two* alignment constraints. The problem here is that the kernel is not reporting the second constraint (region alignment), so we have to make some educated guess.